test.js

```diff
@@ -190,30 +190,30 @@ describe('ns', function() {
 
   describe('metric', function() {
     it('should default to null if not a metric namespace', function() {
-      assert.equal(ns('canadian-things.foods').metric, null);
+      assert.equal(ns('canadian-things.foods').metric, undefined);
```

Comment:
I agree there's little value in regression testing a dropped feature, removed in: https://github.com/mongodb-js/ns/pull/5/commits/f4c1ccc68657d6c965a8adac48473bb9bed5a96b